js/util.js

```diff
@@ -27,17 +27,17 @@ export function title(str) {
  */
```

Comment:
Select list data binding in Vue expects default options to have a value of empty string rather than null. Since all these properties are strings anyway, I converted them all for consistency.